9c-internal/common-dbs/psql-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: psql
+    app.kubernetes.io/instance: common-dbs
+  name: psql-service
+  namespace: common-dbs
+spec:
+  ports:
+    - port: 5432
+      targetPort: 5432
+  selector:
+    app: psql
+  type: ClusterIP
```

Comment:
no need to get access from out of cluster?

Reply:
Right now, I think it would be fine to access it by forwarding since it's an internal DB.
Would it be better to create a service?